test/browser.spec.js

```diff
@@ -6,7 +6,7 @@ const expect = require('chai').expect
 
 describe('browser', () => {
   it('fixtures', () => {
-    const myFixture = loadFixture(__dirname, 'fixtures/test.txt')
+    const myFixture = loadFixture(__dirname, 'test/fixtures/test.txt')
```

Comment:
This would fix the Aegir tests, but wouldn't it make the whole change backwards compatible? This PR would break all packages that are loading fixtures.

Reply:
I know. that is why I put a _warning_ on the first comment. I'll put it bold.

This also removes the need for the other tests to have to check themselves if they are running in Node and passes that responsibility to the loadFixture func https://github.com/ipfs/js-ipfs-api/blob/master/test/name.spec.js#L17-L19